.ci/google-java-format.sh

```diff
@@ -7,7 +7,7 @@ JAR_PATH="$1"
 INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.java" \
     | sed "s|src/it/resources/com/google/checkstyle/test/||" \
     | grep -v "rule711generalform" | grep -v "rule712paragraphs" | grep -v "rule713atclauses" \
-    | grep -v "rule73" \
+    | grep -v "rule734nonrequiredjavadoc" \
```

Comment:
formatter's changes were colliding with our google config for InputInvalidJavadocPosition.java, tests started failing so I had to grep it

Reply:
https://checkstyle.org/google_style.html#a7.3.4

There is InvalidJavadocPosition, yes, formatter might move them.
Ok to keep excluded